# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/cookie/test/browser/setFetchCookie.sjs
# Commit: 01daaabadcd7
# Full Hash: 01daaabadcd7a74a9ab5114b84c5bf06c9583bff
# Author: Tim Huang <tihuang@mozilla.com>
# Date: 2025-01-10 21:44:02
# Regressor Bug: 1915355
# File Overlap Count: 1
# Description:
#   Bug 1915355 - Part 6: Add tests. r=cookie-reviewers,valentin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D223386
# ==============================================================================

diff -r d10ae1bb47a8 -r 01daaabadcd7 netwerk/cookie/test/browser/setFetchCookie.sjs
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/netwerk/cookie/test/browser/setFetchCookie.sjs	Fri Jan 10 08:24:30 2025 +0000
@@ -0,0 +1,15 @@
+function handleRequest(request, response) {
+  response.setStatusLine(request.httpVersion, 200, "OK");
+  response.setHeader(
+    "Set-Cookie",
+    "thirdPartyFetch=value; SameSite=None; Secure;",
+    false
+  );
+  response.setHeader(
+    "Access-Control-Allow-Origin",
+    "https://example.com",
+    false
+  );
+  response.setHeader("Access-Control-Allow-Credentials", "true", false);
+  response.setHeader("Access-Control-Allow-Methods", "GET", false);
+}
